scivision/catalog/data/datasources.json

```diff
@@ -122,6 +122,29 @@
             "ecology",
             "environmental-science"
          ]
+      },
+      {
+         "name":"data-006",
```

Comment:
@acocac I would recommend to give it an informative name, I think the "data-00" convention makes sense for test datasets in the catalog, but otherwise we should drop it

Reply:
Thanks for the suggestion, I'd prefer to keep the data-XX convention rather than user-defined ones... atm this might be the safest route to avoid duplicates... in the long term, we ideally should implement permanent identifiers e.g. [W3ID](https://w3id.org/)

Reply:
@acocac do you think it would help to have an "id" field, separate from "name" so we get the best of both worlds?